components/hab/src/command/studio/enter.rs

```diff
@@ -1,23 +1,27 @@
 use std::{env,
           ffi::OsString,
           fs as stdfs,
-          path::PathBuf};
+          path::{Path,
+                 PathBuf}};
 
 use crate::{common::ui::UI,
             hcore::{crypto::CACHE_KEY_PATH_ENV_VAR,
                     env as henv,
                     fs}};
 
 use crate::{config,
-            error::Result,
+            error::{Error,
+                    Result},
             BLDR_URL_ENVVAR,
             CTL_SECRET_ENVVAR,
             ORIGIN_ENVVAR};
 
 use habitat_core::AUTH_TOKEN_ENVVAR;
+use same_file::is_same_file;
```

Comment:
I'm getting
> error[E0432]: unresolved import `same_file`

when I try this. Is there a change to the cargo.toml that didn't get pushed up, by any chance?

Reply:
:man_facepalming: Fixed!